src/coreclr/jit/morph.cpp

```diff
@@ -11605,16 +11605,16 @@ GenTree* Compiler::getSIMDStructFromField(GenTree*   tree,
 }
 
 /*****************************************************************************
-*  If a read operation tries to access simd struct field, then transform the
-*  operation to the SIMD intrinsic SIMDIntrinsicGetItem, and return the new tree.
-*  Otherwise, return the old tree.
-*  Argument:
-*   tree - GenTree*. If this pointer points to simd struct which is used for simd
-*          intrinsic, we will morph it as simd intrinsic SIMDIntrinsicGetItem.
-*  Return:
-*   A GenTree* which points to the new tree. If the tree is not for simd intrinsic,
-*   return nullptr.
-*/
+ *  If a read operation tries to access simd struct field, then transform the
```

Comment:
did jit-format cause these changes or were these unintentional edits? Could you please try to delete them and run jit-format again?